sdk/state/payment_test.go

```diff
@@ -216,7 +218,7 @@ func TestChannel_ConfirmPayment_acceptsSameObservationPeriod(t *testing.T) {
 		SequenceNumber: int64(202),
 	}
 
```

Comment:
deceiving amount of changes in this file. Replacing putting the channel into an Open State by changing internal data by doing so using external methods instead. Which should prevent having to make these changes more often in the future